packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -309,11 +309,11 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
     }
 
     if (newActiveElement && newActiveElement !== this._activeElement) {
-      this._activeElement = newActiveElement;
-
-      if (isImmediateDescendant) {
-        this._setFocusAlignment(this._activeElement);
+      if (isImmediateDescendant || !this._activeElement) {
+        this._setFocusAlignment(newActiveElement, true, true);
```

Comment:
I'm still not sure about `defaultActiveElement`. I see in constructor that if `defaultActiveElement` prop is present, then `_activeElement` will be set in constructor. As a result, this code wouldn't be triggered. Is that intended when `defaultActiveElement` is used?

Reply:
Also I do see `_activeElement` set to `null` in a couple of places in this module. Wanted to make sure that it won't falsely trigger this code or cause issues...

Reply:
I don't think it will fix defaultActiveElement. Talked in person. We will not address that here and keep this scoped to the manual focus scenario.